Mu2eG4/geomMacros/find_volume_at_point.C

```diff
@@ -0,0 +1,30 @@
+//
+// Useful functions to find a volume at a specific point
+//
```

Comment:
Add the url to the README.md and say that it has important caveats about what happens when a point is on a volume boundary.

Reply:
Done in b52104d39